README.md

```diff
@@ -276,6 +276,25 @@ file, rename and edit it.
 <?php
 
 $settings = array (
+    // Security settings that must be set to avoid insecure SHA-1 usage.
```

Comment:
Right, but we need that block in the basic settings now. We need parts of the security options here, but the advanced options list all security options. So maybe that duplication is fine?